### ISSUE TITLE
[Feature Request] Add MultiStepLR scheduler

### ISSUE THREAD

**Issue by hglee98** [Nota-NetsPresso/netspresso-trainer#558]
### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues.

### Is there any feature that you would like to add?

Add MultiStepLR following pytorch [MultiStepLR](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.MultiStepLR.html)

### Is there any suggestion (or solution) to solve this issue?

- Implement the scheduler
- Register the scheduler
- Add docs

### Will you help us by submitting a PR?

- [X] I will submit a PR about this issue.
